### ISSUE TITLE
Set 'dir' attr on 'html' tag according to current admin language, for RTL support

### ISSUE THREAD

**Issue by remort** [wagtail/wagtail#5263]
### Issue Summary

Add `dir="rtl"` attribute in `html` tag if `LANGUAGE_CODE` is among rtl languages, such as 'he'
The skeleton.html template code could like:

### Steps to Reproduce

1. add `('he', _('Hebrew')),` to `LANGUAGES`
2. add hebrew languages in Settings -> Languages
3. create hebrew page using wagtailtrans plugin, for example root page with slug 'he'
4. view it, so `lang` attr should become 'he'
5. if i have a css for admin interface expecting 'dir="rtl"', this css will not come into power, since no `dir` attribute is set according to current language. this itself a bug. please fix it.
Now I set `dir` attribute with JS-code.

You have a bug for not setting `LANGUAGE_CODE` that us used in skeleton to set `lang` attr.
Bug seems to be closed for wagtail 2.5, so use this version.

**Comment by thibaudcolas** [comment]
Hey @remort, that seems like a useful addition for Wagtail to make its way towards support for RTL language (#1240).

Could you make a list of which languages that would be. It sounds like a good idea to set this based on the defined language instead of having some kind of `USER_INTERFACE_DIR = 'rtl'` settings flag, but then we want the list to be accurate.

I'd also like to know whether this will have any impact on the user interface, if people _don't_ define their own stylesheet? Do browsers behave differently? Or is it just so people defining their custom styles don't have to override the skeleton template?

**Comment by rinti** [comment]
Django's i18n already has support for this. Basically something like:

```html
{% load i18n %}
{% get_current_language_bidi as LANGUAGE_BIDI %}
{% get_current_language as LANGUAGE %}
<html class="no-js" lang="{{ LANGUAGE }}" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}">
```

**Comment by remort** [comment]
@thibaudcolas, yes, 'dir=rtl' applies browser basic RTL styles.

RTL languages by their language code could be found here: https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code

But, as @rinti said, Django can determine direction of current language if i18n is enabled.

**Comment by gasman** [comment]
Fixed in #5273
